### PR TITLE
Partial Filipino Translation - 1 163 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
translating advanced-merging.asc - 1,163 translated words.
code text and computer code text are not counted as translated words.

### CODE REVIEW

File: book/07-git-tools/sections/advanced-merging.asc [hunk 1]
```diff
@@ -351,25 +351,25 @@ end
...
-Perhaps it's not obvious how exactly you should fix this conflict.
-You need more context.
+Talakayin natin ang karamihan sa mga kasangkapan na sa ngayon ay alam mo na iyong magagamit upang malaman kung papaano ang pagkasalungat ay dumating.
+Marahil ito ay hindi halata kung gaano ka eksakto na dapat mong ayusin ang salungat.
+Kailangan mo ng maraming konteksto.
 
-One helpful tool is `git checkout` with the `--conflict' option.
-This will re-checkout the file again and replace the merge conflict markers.
-This can be useful if you want to reset the markers and try to resolve them again.
+Isang matulugin na kasangkapan ay ang `git checkout` kasama ang `--conflict' na opsyon.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``matulugin`` to ``matulungin``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 2]
```diff
@@ -351,25 +351,25 @@ end
...
+Ito ay magre-checkout muli sa file at papalitan ang salungat na merge na mga marka.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``magre-checkout`` to ``mag-checkout muli``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 3]
```diff
@@ -388,25 +388,25 @@ end
...
 [source,console]
 ----
 $ git config --global merge.conflictstyle diff3
 ----
 
-The `git checkout` command can also take `--ours` and `--theirs` options, which can be a really fast way of just choosing either one side or the other without merging things at all.
+Ang `git checkout` na utos ay maaari ding kumuha ng `--ours` at `--theirs` na mga opsyon, na kung saan ay maaaring maging isang tunay na mabilis na paraan sa pagpili pa lamang alinman sa isang panig o sa iba na hindi naka merge sa mga bagay sa lahat.
 
-This can be particularly useful for conflicts of binary files where you can simply choose one side, or where you only want to merge certain files in from another branch - you can do the merge and then checkout certain files from one side or the other before committing.
+Ito ay maaaring maging partikular na kapaki-pakinabang para sa mga salungat na mga binary file na kung saan maaari mo lamang pumili ng isang panig, o kung saan gusto lamang i-merge ang tiyak na mga file mula sa ibang branch - maaari mo ding gawin ang pag-merge at pagkatapos i-checkout tiyak na mga file mula sa isang panig o sa iba bago mag-commit.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: > maaari mo lamang pumili ng isang panig
to
> maaari mo lamang pillin ang isang panig
or
> maaari ka lamang pumili ng isang panig

File: book/07-git-tools/sections/advanced-merging.asc [hunk 4]
```diff
@@ -388,25 +388,25 @@ end
...
 [[_merge_log]]
-===== Merge Log
+===== Pag-merge ng Log
 
-Another useful tool when resolving merge conflicts is `git log`.
-This can help you get context on what may have contributed to the conflicts.
-Reviewing a little bit of history to remember why two lines of development were touching the same area of code can be really helpful sometimes.
+Ang kapaki-pakinabang na kasangkapan kapag naglutas ng merge na mga salungat ay ang `git log`.
+Ito ay nakakatulong sayo na kunin ang konteksto na sa kung ano ang maaaring maiiambag sa mga salungat.
+Pagrepaso nang kaunti sa kasaysayan para tandaan kung bakit ang dalawang mga linya sa development na hinhawakan natin sa parehong nakakatulong minsan.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: * ``hinhawakan`` to ``hinahawakan``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 5]
```diff
@@ -419,10 +419,10 @@ $ git log --oneline --left-right HEAD...MERGE_HEAD
 > c3ffff1 changed text to hello mundo
 ----
 
-That's a nice list of the six total commits involved, as well as which line of development each commit was on.
+Iyan ay magandang listahan sa anim na kabuuang mga commit na kasangkot, pati na rin kung aling linya sa development para sa bawat commit na nangyayari.
 
-We can further simplify this though to give us much more specific context.
-If we add the `--merge` option to `git log`, it will only show the commits in either side of the merge that touch a file that's currently conflicted.
+Maari tayong higit pa na magpasimple upang bigyan ng mas tiyak na konteksto.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``Maari`` to ``Maaari``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 6]
```diff
@@ -431,15 +431,15 @@ $ git log --oneline --left-right --merge
...
-This can be *really* helpful in quickly giving you the context you need to help understand why something conflicts and how to more intelligently resolve it.
+Kung nagpapatakbo ka gamit ang `-p` na opsyon na sa halip, ikaw ay makakuha lamang ng diffs sa file na nagtatapos ng pagkasalungat.
+Ito ay maaaring *talagang* kapaki-pakinabang sa mabilis na pagbibigay sayo ng konteksto na kailangan mo ng tulong na maunawaan kung bakit isang bagay ang mga kasalungat at kung papaano mas matalinong malutas ito.
 
-===== Combined Diff Format
+===== Pinagsamang Diff na Format
 
-Since Git stages any merge results that are successful, when you run `git diff` while in a conflicted merge state, you only get what is currently still in conflict.
-This can be helpful to see what you still have to resolve.
+Dahil ang mga Git ay nag-stage ng anumang mga resulta ng merge na matagumpay, kapag pinatakbo mo ang `git diff` habang nasa pagkasalungat na merge na estado, ikaw ay makakuha lamang kung ano ang kasalukuyang nasa pagkasalungat.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: > makakuha lamang kung ano
to
> makakuha lamang ng kung ano

**Review by heywens — COMMENTED** [review]
Hi Sir @robin-ho, I've suggested few changes for your translation. Kindly check. Thanks.

File: book/07-git-tools/sections/advanced-merging.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: change done

File: book/07-git-tools/sections/advanced-merging.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change, using maaari ka lamang pumili ng isang panig

File: book/07-git-tools/sections/advanced-merging.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

**Comment by mind-blast** [comment]
Done change all suggested . Thank you very much for the proof read. =)

**Pull request merged**